server/forms.py

```diff
@@ -0,0 +1,65 @@
+from django import forms
+from .models import User, Player, Membership, Vaccination
+
+
+class UserForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ["email", "phone"]
+
+
+class PlayerForm(forms.ModelForm):
+    date_of_birth = forms.DateField(widget=forms.DateInput(attrs={"type": "date"}))
+
+    class Meta:
+        model = Player
+        fields = [
+            "first_name",
+            "last_name",
+            "date_of_birth",
+            "gender",
+            "city",
+            "state_ut",
```

Comment:
Can this be empty? Otherwise we need to add an option for "Not in India".

Reply:
changed this to have `blank=True` for `state_ut` in the `models.py`